flaskbb/utils/helpers.py

```diff
@@ -105,16 +108,21 @@ def do_topic_action(topics, user, action, reverse):
     if not topics:
         return False
 
-    from flaskbb.utils.requirements import (IsAtleastModeratorInForum,
-                                            CanDeleteTopic, Has)
+    from flaskbb.utils.requirements import (
+        IsAtleastModeratorInForum,
+        CanDeleteTopic,
+        Has,
+    )
 
     if not Permission(IsAtleastModeratorInForum(forum=topics[0].forum)):
-        flash(_("You do not have the permissions to execute this "
-                "action."), "danger")
+        flash(
+            _("You do not have the permissions to execute this " "action."),
```

Comment:
10/10 code reformatting :stuck_out_tongue:

Reply:
Filed under "odd things black does when the code is already a little odd"